stonesoup/updater/wrapper.py

```diff
@@ -0,0 +1,23 @@
+from . import Updater
+from ..base import Property
+from ..types.detection import GaussianDetection
+
+
+class DetectionAndTrackSwitchingUpdater(Updater):
```

Comment:
Docs

Reply:
```suggestion
class DetectionAndTrackSwitchingUpdater(Updater):
"""Updater which sorts between Detections and Tracks"""
```
Does this look alright?